### ISSUE TITLE
Repose alters behavior of unrelated blocks

### ISSUE THREAD

**Issue by gchpaco** [delvr/Repose#13]
Anvils, despite being in neither the granular blocks nor natural stone blocks list, will spread when they fall, contrary to vanilla behavior and any expectation.  As it happens, Rustic chandeliers share some falling logic with anvils, and with repose installed cannot be placed on ropes or chains.  Attempting to will cause the chandelier to immediately fall (and spread on falling) instead of behaving as expected.

There is an entry in the repose.properties file:
```
# If true, non-soil blocks that can spread will also trigger avalanches when disturbed. Note: this option has no effect
# when using the EnviroMine mod.
repose.avalanches=false
```

What determines "non-soil blocks that can spread" ? There isn't any such list in the config file, nor any visible setting to allow/disallow the behavior of non-soil blocks spreading.

Setting this to false has no impact on either anvils or chandeliers. Setting `repose.blockSpread=false` does prevent anvils and chandeliers from spreading, but it also prevents everything else in the mod from spreading.

Even with both settings off, rustic chandeliers still cannot be placed; they drop immediately.

Minecraft 1.12.2, rustic 0.4.3, repose 1.12-1.5, forge 2512.

**Comment by delvr** [comment]
This is fixed in the new Repose 1.12-1.5.1. Please give it a try, thanks!

**Comment by gchpaco** [comment]
That did it!  Thanks.

-- 
Graham Hughes
http://www.sigwinch.org/Graham
